learning/katas/go/windowing/adding_timestamp/pardo/task.md

```diff
@@ -0,0 +1,43 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+# Assigning Timestamps
+
+A DoFn can assign timestamps to elements of a PCollection.  We will see the use these timestamps in the next lesson
```

Comment:
Based on your understanding of the Kata form, should the purpose of adding timestamps be mentioned in this lesson or does that wait for a subsequent lesson?

Reply:
Following the pattern of the existing katas for Java, I made a brief mention of the purpose but leave the subsequent lesson to address it's use and purpose in windowing.

Reply:
SGTM Thanks!